packages/codemirror/src/Editor.ts

```diff
@@ -9,12 +9,13 @@ export interface IPosition {
 }
 
 export class Editor extends HTMLWidget {
-    private _codemirror: CodeMirror.EditorFromTextArea;
+    protected _codemirror: CodeMirror.EditorFromTextArea;
```

Comment:
This change (which is not needed) is causing the build failure.

Reply:
I noticed that too, had it changed to "protected" to make debugging easier. I've rolled it back and amended.